packages/mermaid/src/docs/intro/index.md

```diff
@@ -106,7 +106,7 @@ Class01 : int gorilla
 Class08 <--> C2: Cool label
 ```
 
-### Git graph
+### [Git graph](../syntax/gitgraph.md)
```

Comment:
I added this to make it consistent with how all other chart types are documented.